components/service/experiments/src/main/java/mozilla/components/service/experiments/Experiments.kt

```diff
@@ -47,34 +48,39 @@ open class ExperimentsInternalAPI internal constructor() {
      * @param applicationContext [Context] to access application features, such
      * as shared preferences.
      */
-    fun initialize(
+    open fun initialize(
```

Comment:
Why is this function `open`? I can't seem to find a place in this PR in which it is overriden. Are you planning a follow-up PR for using this?